pandas/core/series.py

```diff
@@ -4630,6 +4630,34 @@ def isin(self, values) -> "Series":
         4     True
         5    False
         Name: animal, dtype: bool
+        
+        New behaviour from pandas v1.2.0 caused some tests to fail because users where unaware of this change.
```

Comment:
I don't think that we need an example from before 1.2 in the docstrings. Just a comment that we do no longer match strings containing numbers with integers and an example from the new behavior? 

Also pls use versionchanged

Reply:
Yeah, I was not sure if I should add the before part or not. Ok, I will remove it.
What do you mean by use versionchanged?

Reply:
.. versionchanged:: 0.24.0

Reply:
Ok, thanks for the help!